src/node/stage/approved.rs

```diff
@@ -1665,28 +1590,51 @@ impl ApprovedPeer {
                     continue;
                 }
             };
-            match self.send_signed_message(&msg) {
+            match self.send_signed_message(core, &msg) {
                 Ok(()) => trace!("Resend {:?}", msg),
                 Err(error) => debug!("Failed to resend {:?}: {:?}", msg, error),
             }
         }
     }
 
+    // Send message over the network.
+    pub fn send_signed_message(&mut self, core: &mut Core, msg: &MessageWithBytes) -> Result<()> {
+        let (targets, dg_size) = self.chain.targets(msg.message_dst())?;
+
+        trace!("Sending {:?} via targets {:?}", msg, targets);
```

Comment:
maybe having this log after the filter collection?

Reply:
:+1: In a followup PR though.